src/interfaces/useCase.ts

```diff
@@ -2,11 +2,12 @@ import UseCaseResponse from '../classes/UseCaseResponse';
 import TelegramMessage from '../classes/TelegramMessage';
 
 interface UseCase {
-  readonly useCaseName: string;
+  readonly name: string;
```

Comment:
Enum over String in TS im static umfeld

Reply:
Wie meinst du?

Reply:
use case names sind ja fest, dann willst du dafür keine Strings sondern enums nutzen

Reply:
Das ist der Name wie er z. B. im Dashboard angezeigt wird, also der Anzeigename, nicht der interne Name. Und wenn wir dafür ein Enum erzeugen, sind die Infos für einen Use Case nicht mehr alle an einem Ort, und wir haben ein extra Enum.
So habe ich mir das vorgestellt:
https://github.com/JaydenUlysses/Olaf-the-coolest-of-Bots/blob/32fbaa3df6e76f3f6b7ffcd12e0db7fde32cfcf9/src/usecases/imageoftheday/imageofthedayUsecase.ts#L15-L17